corelib-definitions/src/main/java/eu/europeana/corelib/definitions/model/facets/inverseLogic/CommonPropertyExtractor.java

```diff
@@ -0,0 +1,794 @@
+package eu.europeana.corelib.definitions.model.facets.inverseLogic;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CommonPropertyExtractor {
+
+    private final static Map<Integer, String> mimeTypes = new HashMap<>();
```

Comment:
Performance wise it's better to always state a minimum size of you know a map is gonna be big. In this case you actually know the exact size you need, give that is the only parameter of the HashMap constructor.
 .... = new HashMap<>(776);